src/drive/ducks/services/lib.js

```diff
@@ -0,0 +1,6 @@
+/* global cozy */
+
+export const getFileDownloadUrl = async (id) => {
```

Comment:
I think this helper already exists in the `files` duck, it'd be better to not have this one here

Reply:
Aah good catch, there's *almost* the same thing in the general actions file. I'll use that one instead.